### DISCUSSION TITLE
Change PREVIOUS/NEXT anchor links value

### DISCUSSION THREAD

**Discussion by SaidRasinlic** [ddnexus/pagy#705] [Q&A] [answered]
Hi everyone,

I hope you're all doing well! I'm currently working with Pagy Rails pagination and looking for a way to personalize the 'Previous' and 'Next' link text. I've tried a few approaches but haven't found a straightforward solution yet.

When I use <%== pagy_nav(`@pagy`) %>, it generates a nav, but the anchors are showing only arrows left and right without 'previous' and 'next' text. So I want to add text besides the arrows.

For example in my pagy.rb I tried all of these but unsuccessfully

Pagy::DEFAULT[:link_prev] = { text: 'Custom Previous' }
Pagy::DEFAULT[:prev] = '&laquo; Custom Previous'
Pagy::DEFAULT[:pagy_prev] = 'Previous'
Pagy::I18n.t['pagy.prev'] = 'Custom Previous'

In my view I also tried  <%== pagy_nav(`@pagy`, prev: { text: "Previous" }) %> but still unsuccessfully..

Does anyone have suggestions or tips on how I can easily change the text for the 'Previous' and 'Next' links in Pagy pagination? Do I have to create custom pagination template, which would not be preferred?

Your insights would be greatly appreciated! Thank you in advance for your help.

**Comment by SaidRasinlic** [comment, marked as answer]
I made it work by going to projects locales/en.yml and just adding:
```
pagy:
   prev: "&lt; Previous"
   next: "Next &gt;"
```

To check all locales this link will be out of help https://github.com/ddnexus/pagy/blob/master/gem/locales/en.yml

However, it would be dope if we could override it somehow per pagy instance or in pagy.rb (I'm new to Pagy; maybe it can; if you know, please let us newbies know too).

Cheers 😃

**Reply by ddnexus** [reply]
> it would be dope if we could override it somehow per pagy instance or in pagy.rb

It would also be inefficient and messy.

**Reply by pramodtech** [reply]
I am having same issue. My projects en.yml is located at `config/locales/` folder. I updated file as mentioned in this answer but it doesn't work.
My application.rb file has
```
config.i18n.default_locale = :en
config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}').to_s]
```

**Reply by pramodtech** [reply]
Never mind, I missed adding `require 'pagy/extras/i18n'`.
